launchpad-frontend/src/index.js

```diff
@@ -16,7 +16,7 @@ export const pages = {
   landing: "/landing",
   notifications: "/notifications",
   applications: "/applications",
-  jobs: "/jobs",
+  jobs: "/jobs/:jobId",
```

Comment:
Is there a way to have the Job Postings page default as `/jobs` instead of `/jobs/:jobId` if there is no job posting selected?

Reply:
Good point! It'll be better to have it display as /jobs instead when nothing is selected. Updated :)